templates/advantage/_stripe-modal.html

```diff
@@ -46,18 +33,40 @@ <h2 class="p-modal__title js-contract-name u-no-margin--bottom" id="modal-title"
 
           <div class="p-form__group p-form-validation row u-no-padding u-vertically-center">
             <div class="col-3">
-              <label class="u-no-padding--top" for="name">Name on card:</label>
+              <label class="u-no-padding--top" for="email">Email my receipt to:</label>
             </div>
 
             <div class="col-9">
-              <input class="p-form-validation__input" name="name" type="text" required value="{{ user_info.fullname }}" />
+              <input class="p-form-validation__input" name="email" type="email" required value="{{ user_info.email }}" />
             </div>
 
             <div class="col-9 col-start-large-4">
               <span class="p-form-validation__message u-hide"></span>
             </div>
           </div>
 
+          {% if account is defined %}
+            <div class="p-form__group p-form-validation row u-no-padding u-vertically-center">
+              <div class="col-3">
+                <label class="u-no-padding--top" for="name">Account name:</label>
+              </div>
+
+              <div class="col-9">
+                <input class="p-form-validation__input" name="name" type="text" required value="{{ account.name }}"{% if account %}disabled{% endif %} />
+              </div>
+
+              {% if not account %}
+                <div class="col-9 col-start-large-4">
+                  <small>For example, your organization ot team name</small>
```

Comment:
```suggestion
                  <small>Your name, or the name of your organisation.</small>
```
(Canonical uses UK spelling, so we need to say “organisation”.)

Reply:
Done.